packages/webdriverio/src/utils/findStrategy.js

```diff
@@ -44,6 +44,10 @@ const defineStrategy = function (selector) {
     if (selector.startsWith('*=')) {
         return 'partial link text'
     }
+    // Use id strategy if the selector starts with id=
+    if (selector.startsWith('id=')) {
```

Comment:
Could you share why do you want to add it? If more common would be use `#` symbol.

Reply:
The reason for this is when using the `#` symbol, findStrategy uses the css selector to look for the element using id. The youi tv appium driver does not allow using css selector but allows id. Currently, webdriver io does not use id selectors with the $ and $$ function. 

We can change the default id selector to use `#` but I think it would still be better to have both so people can still use `#` to look for id using css selectors and `id=` to look for id using id selector.